tests/MongoFramework.Tests/MongoFramework.Tests.csproj

```diff
@@ -20,19 +20,18 @@
   </ItemGroup>
 
   <Choose>
-    <When Condition=" '$(TargetFramework)' == 'net5.0' ">
+    <When Condition=" '$(TargetFramework)' == 'netcoreapp2.1' ">
       <ItemGroup>
-        <PackageReference Include="System.Linq.Async" Version="5.0.0" />
+        <PackageReference Include="System.Linq.Async" Version="4.1.1" />
       </ItemGroup>
     </When>
     <Otherwise>
       <ItemGroup>
-        <PackageReference Include="System.Linq.Async" Version="4.1.1" />
+         <PackageReference Include="System.Linq.Async" Version="5.1.0" />
       </ItemGroup>
     </Otherwise>
```

Comment:
This change doesn't look to do what you think it does - before anything not .NET 5 was using `System.Linq.Async` 4.1.1 though your change now says only .NET Core 2.1 will use `System.Linq.Async` 4.1.1.

I believe these checks were here because of dependency issues with Azure Functions. Really though, if we can run the latest version on all targets, that will be better than running specific versions on individual targets.

Reply:
Admittedly, we shouldn't actually need this block anymore as the `MongoFramework.Tests` project depends on `MongoFramework` which define these dependencies anyway.

Reply:
Removed